api/pkg/filters/auth/auth.go

```diff
@@ -35,28 +34,27 @@ func NewNoAuth() AuthBase {
 
 func (auth *NoAuth) Filter(req *restful.Request, resp *restful.Response, chain *restful.FilterChain) {
 	ctx := req.Attribute(c.KContext).(*c.Context)
+	log.Log(ctx.TenantId)
 	params := req.PathParameters()
 	if tenantId, ok := params["tenantId"]; ok {
 		ctx.TenantId = tenantId
 	}
 
-	log.Log("auth filter", ctx.TenantId)
 	ctx.IsAdmin = ctx.TenantId == c.NoAuthAdminTenantId
 	chain.ProcessFilter(req, resp)
 }
 
 func FilterFactory() restful.FilterFunction {
 	var auth AuthBase
-	log.Log("authStrateger:\n")
-	log.Log(os.Getenv("AUTH_AuthStrategy"))
-	switch os.Getenv("AUTH_AuthStrategy") {
+	s := cf.GetAuthStrateger()
```

Comment:
I haven't seen the update, it should be `GetAuthStategy`

Reply:
> I haven't seen the update, it should be `GetAuthStategy`
   I Have update it, thanks for your suggestion